Hallo.Giraffe.Sample/Startup.fs

```diff
@@ -0,0 +1,78 @@
+namespace Hallo.Giraffe.Sample
+
+open System.Text.Encodings.Web
+open System.Text.Json
+open System.Text.Json.Serialization
+open Hallo
+open Hallo.AspNetCore
+open Hallo.AspNetCore.Sample
+open Hallo.AspNetCore.Sample.Data
+open Hallo.AspNetCore.Sample.Models
+open Hallo.AspNetCore.Sample.Models.Hypermedia
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.AspNetCore.Http
+open Microsoft.Extensions.DependencyInjection
+open Giraffe
+open FSharp.Control.Tasks
+
+type Startup() =
+
+    let getPeopleHandler: HttpHandler =
+        fun (next: HttpFunc) (ctx: HttpContext) ->
+            task {
+                let peopleRepo =
+                    ctx.RequestServices.GetRequiredService<PeopleRepository>()
+
+                let model = peopleRepo.List(Paging())
+                do! HalJsonGenerator.HalHandler(ctx, model)
```

Comment:
Similar to the ASP.NET Core sample, this F# sample app calls the json generator which writes to the response and returns